pkg/config/server/gitlab.go

```diff
@@ -13,6 +13,9 @@ type GitLabConfig struct {
 	// events to.
 	WebhookSecrets `mapstructure:",squash"`
 
+	// ExternalWebhookURL is the URL that we will send our webhook to
+	ExternalWebhookURL string `mapstructure:"external_webhook_url"`
```

Comment:
Do we need the full url here? Seems error prone to me. What's missing for us to leverage the existing functionality which sets the provider class to have its own path?

Reply:
When we did that in staging, it was easy to mis-configure the existing setting to point to the `.../github` path (previous configuration setting), and have all the URLs look like `.../github/gitlab`, and get delivered to the wrong endpoint.

It seemed to me that explicit (and being able to find the actual URL from the config for debugging) was clearer than implicit re-use of an existing value with new meaning.